content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Support validating certificate revocation with [Online Certificate Status Protocol](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol) (OCSP)
```

Comment:
nit:
I'm not sure if this can bring us any issue, but if I remember correctly, the content from wikipedia can be changed by anyone anytime? So may be we should post the corresponding RFC here, just to be safe?
```suggestion
- Support validating certificate revocation with [Online Certificate Status Protocol](https://datatracker.ietf.org/doc/html/rfc6960) (OCSP)
```